b2

```diff
@@ -1113,7 +1113,9 @@ class StoredAccountInfo(AbstractAccountInfo):
     DOWNLOAD_URL = 'download_url'
 
     def __init__(self):
-        self.filename = os.path.expanduser('~/.b2_account_info')
+        account_info_filename = os.environ['B2_ACCOUNT_INFO']
```

Comment:
If `B2_ACCOUNT_INFO` is not exported, this will raise an exception. Maybe it could be done like this:

```python
user_account_info_path = os.environ.get('B2_ACCOUNT_INFO')
self.filename = user_account_info_path or os.path.expanduser('~/.b2_account_info')
```

?